app/test/unit/find-forked-remotes-to-prune-test.ts

```diff
@@ -0,0 +1,126 @@
+import { findForkedRemotesToPrune } from '../../src/lib/stores/helpers/find-forked-remotes-to-prune'
+import { Branch, BranchType } from '../../src/models/branch'
+import { CommitIdentity } from '../../src/models/commit-identity'
+import { GitHubRepository } from '../../src/models/github-repository'
+import { PullRequest } from '../../src/models/pull-request'
+import { IRemote } from '../../src/models/remote'
+import { gitHubRepoFixture } from '../helpers/github-repo-builder'
+
+function createSamplePullRequest(
+  gitHubRepository: GitHubRepository,
+  userName: string,
+  baseBranchName: string
+) {
+  return new PullRequest(
+    new Date(),
+    'desktop',
+    1,
+    {
+      ref: 'refs/heads/master',
```

Comment:
This won't matter for these tests but just in case this gets reused the format for the head and base ref is not qualified (i.e. it's 'main' rather than 'refs/heads/main'). And we try to avoid using `master` in new code (see https://github.com/desktop/desktop/pull/10833) (I know this was just copy-pasted from elsewhere but we might as well clean it up along the way).

```suggestion
      ref: 'main',
```